requirements/constraints.txt

```diff
@@ -11,9 +11,14 @@
 # Common constraints for Open edX repos
 -c https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt
 
-# Keep same platform version
-pylint<3.0
-pycodestyle<2.9.0
+# Versions >= 3.3.0 drop support for Python 3.8
+# Ensures a consistent pylint version across all Python environments to avoid triggering varying warnings
+pylint<3.3.0
+
+# Date: 2023-07-31
```

Comment:
Thank you for considering my suggestion! The most relevant parts of this format are the description and the issue, if it exists and any reference to when to unpin the constraint. So I think we can drop the date. We could still do a git blame to check when it was introduced.